projects/core/src/dropdown/dropdown.stories.mdx

```diff
@@ -9,6 +9,14 @@ import { Meta, Props, Story, Preview } from '@web/storybook-prebuilt/addon-docs/
     This component or utility is offered as a preview. This means we are currently working on it and seeking feedback.
     Please be aware that this component or utility may have breaking changes before we finish working on it.
   </cds-alert>
+  <cds-alert>
+    <strong>Breaking:</strong> In 6.2.0, the `popup` attribute on the dropdown trigger was renamed to `aria-controls` to
```

Comment:
Breaking changes need a major version bump, this looks like a minor version bump.

Reply:
It is a breaking change, but we also clearly spell out that the component is in beta and may break. I would prefer not to hold this til a next major version.

Reply:
It your call, I'm not a maintainer any more :) 
It will break all of my apps tho.